javalib/src/main/scala/java/util/package.scala

```diff
@@ -28,6 +38,21 @@ package object util {
       Objects.hashCode(inner)
   }
 
+  private[util] final case class IdentityBox[+K](inner: K) {
```

Comment:
This can be kept private in the companion object of `IdentityHashMap`. No need to make it more accessible.

Reply:
Moved this as well.